### PR TITLE
Cleanup docs for bpfctl load rework

### PR DESCRIPTION
**Pull request by Billy99**
In tutorial.md, there were still references to --path and --from-image. Also, to make the webpage easier to read, when linking other pages, reference the page title, not the file name.

Signed-off-by: Billy McFall <22157057+Billy99@users.noreply.github.com>

### CODE REVIEW

**Review by anfredette — COMMENTED** [review]
All your changes LGTM.  I just have a couple of comments unrelated to your changes, which could be addressed now or later.

You don't need to do this now, but I think It's useful to have your user in the bpfd user group whether you're manually starting bpfd or running it as a service.  I normally start bpfd manually during development, but I have the target/debug dir in my PATH and my user in the bpfd group, so I don't need to use sudo or the full path for bpfctl.  So, it seems like this could be described separately in the tutorial (or somewhere).

File: docs/tutorial.md [hunk 1]
```diff
@@ -14,7 +14,12 @@ There are several ways to launch and interact with `bpfd` and `bpfctl`:
...
 This tutorial uses examples from the [xdp-tutorial](https://github.com/xdp-project/xdp-tutorial).
-You will need to check out the git repository and compile the examples.
+Use either pre-built container images from
+[https://quay.io/organization/bpfd-bytecode](https://quay.io/organization/bpfd-bytecode) or
+check out the [xdp-tutorial](https://github.com/xdp-project/xdp-tutorial) git repository and
+compile the examples.
+[EBPF Bytecode Image Specifications](./shipping-bytecode.md) describes how BPF bytecode is
+packaged in container images.
 
 ### Step 1: Build `bpfd`
```
[the remark this answers is not in the document]
Comment (anfredette) [reply docs/tutorial.md:24]: > I think we should add a note here or at the bottom of this section that says: "If this is your first time using bpfd, follow the instructions in [Setup and Building bpfd](./building-bpfd.md).

On second thought, I think we should make this change now, but the other one can wait.

**Comment by Billy99** [comment]
> I think It's useful to have your user in the bpfd user group whether you're manually starting bpfd or running it as a service.
So the issue is that `./scripts/setup.sh` has the logic to create the `bpfd` user and user group. And that hasn't been run yet. It is now only run to setup bpfd to run as a service.

File: docs/tutorial.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Billy99) [reply docs/tutorial.md:24]: Done

**Review by anfredette — APPROVED** [review]
/LGTM

**Pull request merged**